packages/composables/src/getters/categoryGetters.ts

```diff
@@ -1,99 +1,20 @@
-import { CategoryGetters, AgnosticCategoryTree, AgnosticBreadcrumb } from '@vue-storefront/core';
 import type { Category, CategoryDetails } from '@vue-storefront/plentymarkets-api';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-function getTree(category: Category): AgnosticCategoryTree {
-  return {
-    label: getCategoryDetails(category.details)?.name || '',
-    slug: getCategoryDetails(category.details)?.nameUrl || '',
-    items: category.children ? category.children.map(cat => getTree(cat)) : [],
-    isCurrent: false,
-    count: category?.itemCount[0]?.count || 0
-  };
+interface categoryGetters {
+  getCategoryName(category: Category),
+  getCategoryDetails(category: Category)
```

Comment:
```suggestion
  getCategoryName(category: Category): string,
  getCategoryDetails(category: Category): CategoryDetails | null
```
**todo**: Add return types.

Reply:
done